goinstant/docker.go

```diff
@@ -186,4 +193,166 @@ func RunDockerCommand(commandSlice ...string) {
 		fmt.Fprintln(os.Stderr, "Error waiting for Cmd.", stderr.String(), err)
 		return
 	}
+
+	if commandName == "git" {
+		if len(commandSlice) < 2 {
+			fmt.Fprintln(os.Stderr, "Not enough arguments for git command", stderr.String(), err)
+			return
+		}
+		pathToPackage = commandSlice[1]
+		// Get name of repo
+		urlSplit := strings.Split(pathToPackage, ".")
+		urlPathSplit := strings.Split(urlSplit[len(urlSplit)-2], "/")
+		repoName := urlPathSplit[len(urlPathSplit)-1]
+
+		pathToPackage = filepath.Join(".", repoName)
+	}
+	return
+}
+
+func mountCustomPackage(deployEnvironment string, pathToPackage string) {
+	gitRegex := regexp.MustCompile(`\.git`)
+	httpRegex := regexp.MustCompile("http")
+	zipRegex := regexp.MustCompile(`\.zip`)
+	tarRegex := regexp.MustCompile(`\.tar`)
+
+	if gitRegex.MatchString(pathToPackage) {
+		pathToPackage = runCommand("git", []string{"clone", pathToPackage}...)
+	} else if httpRegex.MatchString(pathToPackage) {
+		resp, err := http.Get(pathToPackage)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, "Error in dowloading custom package", err)
```

Comment:
Should change these errors to Panic to prevent the method from returning an empty path and causing unexpected behaviour to the cli user